internals/generators/organization/config.yml.hbs

```diff
@@ -3,6 +3,7 @@ name: "{{name}}"
 slug: "{{slug}}"
 settings:
   mobile_phone_verification: {{mobile_phone_verification}}
+  identity_verification: {{identity_verification}}
```

Comment:
if we can simplify as suggested in the previous comment, this will not be necessary